### ISSUE TITLE
A new benchmark experiment for clarifying what is the actual SOTA

### ISSUE THREAD

**Issue by LifeIsStrange** [Nixtla/statsforecast#198]
Thanks to your excellent benchmarking, you show that classical methods such as ETS outperform neuralprophet and arima outperforms fb-prophet.

Reading your analysis:https://github.com/Nixtla/statsforecast/tree/main/experiments/neuralprophet

While 
> NeuralProphet bridges the gap between traditional time-series models and deep learning methods.

is shown to be false 
> are there reliable deep learning or hybrid deep+classical methods that outperforms classical methods in accuracy? 

This is the useful question that is left unanswered.

In it you share this repository https://github.com/nixtla/neuralforecast

Of which one can find this implementation, which is very interesting for two reasons:
https://www.sciencedirect.com/science/article/pii/S0169207019301153
1) it is the winner of the M4 competition.
2) it is a hybrid that actually use ETS but goes beyond it.
The question is are there cases where this winner perform worse than regular ETS.

The second thing to test would be the winner of the M5 competition,
It seems most winning implementations used lightGBM, which is not a neural network and yet is a deep learning method.
This is an elephant in the room that is left uncompared.
https://www.sciencedirect.com/science/article/pii/S0169207021001874
